### PR TITLE
Update kind to 0.9.0

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

**Comment by kaweue** [comment]
recheck

**Comment by kaweue** [comment]
recheck ha

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This is first check-in to R2005
 [Deployment on Openshift 4 and KVM](deploy/openshift/docs/Openshift-KVM.md)
 
 ## Requirements
-  * Go 1.13
+  * Go
```
Comment (katrybacka) [inline README.md:11]: maybe minimal version at least?

Comment (katrybacka) [reply README.md:11]: minimal supported

**Review by katrybacka — APPROVED** [review]

**Review by katrybacka — CHANGES_REQUESTED** [review]
i made a mistake and approved instead of request changes xD

File: README.md [hunk 2]
```diff
@@ -45,17 +45,17 @@ We use Goland and Visual Studio Code. Install your favourite one.
...
     
     $ kind version
-    kind v0.7.0 go1.13.8 darwin/amd64
+    kind v0.9.0 go(...) darwin/amd64
 
 Verify if it works (Linux):
     
     $ kind version
-    kind v0.7.0 go1.13.8 linux/amd64
+    kind v0.9.0 go(...) darwin/amd64
```
[the remark this answers is not in the document]
Comment (kaweue) [reply README.md:58]: ok I will fix that

File: README.md (hunk 1 above)
Comment (kaweue) [reply README.md:11]: I will change that

**Review by katrybacka — APPROVED** [review]

**Review by kabart — APPROVED** [review]

**Pull request merged**